src/components/Box/Box.tsx

```diff
@@ -121,7 +125,8 @@ const Box: VibeComponent<BoxProps> & {
       paddingBottom,
       paddingStart,
       textColor,
-      backgroundColor
+      backgroundColor,
```

Comment:
Note to ourselves because I keep forgetting: let's add trailing comma for out prettier/lint 😅

Reply:
Yes definitely